packages/modules/tl/src/components/SearchApplication.js

```diff
@@ -89,19 +89,7 @@ const SearchApplication = ({tenantId, t, onSubmit, data }) => {
         {
             Header: t("TL_LICENSE_NUMBERL_LABEL"),
             disableSortBy: true,
-            Cell: ({ row }) => {
-                return (
-                    <div>
-                        {row.original["licenseNumber"] ?
-                            <span className="link">
-                                <Link to={`/digit-ui/employee/tl/application-details/${row.original["applicationNumber"]}`}>
-                                    {row.original["licenseNumber"]}
-                                </Link>
-                            </span>
-                            : "-"}
-                    </div>
-                );
-            },
+            accessor: (row) => GetCell(row.licenseNumber || "-"),
```

Comment:
is navigation to application details is handled?

Reply:
no cant navigate to license number as application details page takes application number